src/server/db/migrations/20160217083000-simsupplier-migration.test.js

```diff
@@ -38,18 +38,12 @@ module.exports.up = function(db, config)
 
     // -----
 
-    // Load data.
-    const user2SupplierData = require(path + '/user2supplier.json');
```

Comment:
So if I understand you correctly @gr4per we leave it as it is for now? This file actually does not do any migrations, but data population.